docs/networking.md

```diff
@@ -0,0 +1,156 @@
+# Networking
+
```

Comment:
I think I would start with something more basic, sort of:

```
By default, linuxkit recognizes network devices but does not configure them. You must provide either static configuration or dhcpd, onboot or as a service, to get an IP.

The normal and simplest use case is just to include dhcpd in onboot services <include example here from linuxkit.yml>. This will cause dhcpd to startup *once* and assign an IP.

The alternative method is to make it a service. This ensures dhcpd catches any updates, but may cause issues with services that expect an IP address on startup, since service launch order is *not* guaranteed.

Similarly, using WiFi requires wpa_supplicant in addition to dhcpd.

Finally, if you require any specialized firmware or device drivers for your network card, you may need to load it.

While the common setup .... <continue from here>
```